ch32fun/ch32fun.mk

```diff
@@ -183,6 +183,27 @@ else
 		endif
 
 		TARGET_MCU_LD:=3
+	else ifeq ($(findstring CH59,$(TARGET_MCU)),CH59) # CH592 1
+		TARGET_MCU_PACKAGE?=CH592F
+		CFLAGS_ARCH+=-march=rv32imac_zicsr \
```

Comment:
zicsr should not be specified here, because... older GCC's.  Specify it inline, based on the `ADD_ARCH_ZICSR` macro.

Reply:
I had this in some macro like that at first, but then it did not work for some reason. I'll try again

Reply:
actually removing `_zicsr` works fine now, so that's an easy fix

Reply:
[ea55eee](https://github.com/cnlohr/ch32fun/pull/534/commits/ea55eeebf7c5cb382c77407d0791805015001375)